client/vscode/src/chatActivation.ts

```diff
@@ -48,6 +54,22 @@ export function registerChat(languageClient: LanguageClient, extensionUri: Uri)
                         })
                     })
                 break
+            case feedbackNotificationType.method:
+                languageClient.sendNotification(feedbackNotificationType, message.params)
+                break
+            case followUpClickNotificationType.method:
+                if (!isValidAuthFollowUpType(message.params.followUp.type))
+                    languageClient.sendNotification(followUpClickNotificationType, message.params)
```

Comment:
the `else` clause in a real world use case is that the extension handles the auth right?

Reply:
yup, correct